application/models/user.go

```diff
@@ -103,6 +106,9 @@ func (u *User) CreateAccessToken(org Organization, clientID string) (string, int
 }
 
 func (u *User) GetOrgIDs() []interface{} {
```

Comment:
What is the reason/benefit of returning an array of interfaces rather than a concrete type?

Reply:
I believe it's for the convenience of calling it from a `Where()`, like: